#wheel_width: The width of the wheels
#wheel_radius: The radius of the wheels
#frequency: publishing frequencies of "real_world" messages
#wheel_base: The distance between the centers wheels
#chassis_length: The length of the main chassis link
#chassis_thickness: The thickness of the plate that forms the chassis
#rectangular_obstacles_length: length of a rectangular obstacles' side along its positive x axes
#rectangular_obstacles_width: length of rectangular obstacles' side perpendicular to its positive x axes
#rectangular_obstacles_orientation: angle between the rectangular obstacles' positive x axes and the world frame's positive x axes
#circular obstacles: x, y are coordinates of the obstacles' centers in the map frame.
#rectangular obstacles: x, y are coordinates of the obstacles' centers in the map frame.
#orientation is the angle between the map's positive x axis, and the obstacle's positive x axis (obstacle frame is located at itscenter)
#length is defined as the length of the side that is parallel to the x axis of the cube. Width is parallel to the y axis of the cube. Height is automatically 0.5m
#laser_frequency: update freuqency of the laser scanner
#scan_radius: maximum scan radius of the laser scanner

#miu_vel: mean of velocities [angular, linear]
#stddev_vel: standard deviation of velocity
#miu_z:  mean of observation, [range, bearing]
#stddev_z:  standard deviation of observation


wheel_radius: 0.2
wheel_base: 1.2
frequency: 60

odom_frame_id: 'odom'
body_frame_id: 'base_link'
map_frame_id: 'map'
world_frame_id: 'world'
actual_robot_frame_id: "actual_robot"

left_wheel_joint: 'left_wheel_axle'
right_wheel_joint: 'right_wheel_axle'

miu_vel: [0.0, 0.0]
stddev_vel: [0.0,0.0]
miu_z:  [0.0,0.0]
stddev_z:  [0.0,0.0]

circular_obstacles_x: [3.0, 2.0, 1.0]
circular_obstacles_y: [3.0, 3.0, 3.0]
circular_obstacle_radius: 0.2    #assume cylindrical circular_obstacles
rectangular_obstacles_x: [0.0, 0.0, 4.1, -4.1]
rectangular_obstacles_y: [-4.1, 4.1, 0.0, 0.0]
rectangular_obstacles_length: [8.4, 8.4, 8.0, 8.0]
rectangular_obstacles_width: [0.2, 0.2, 0.2, 0.2]
rectangular_obstacles_orientation: [0.0, 0.0, 1.57, -1.57]
laser_frequency: 40
scan_radius: 3.0

